src/components/Facilitator/AddHotelForm.js

```diff
@@ -152,16 +152,16 @@ export default function AddHotelForm() {
   return (
 
     // class="p-3 bg-indigo-400 text-white w-full hover:bg-indigo-300"    
-    <div className="h-screen  overflow-hidden flex items-center justify-center bg-gray-400 ">
-      <div className="leading-loose">
+    <div className="overflow-hidden flex items-center justify-center bg-gray-400 ">
```

Comment:
If you want to allow vertical scroll
```suggestion
    <div className="overflow-x-hidden flex items-center justify-center bg-gray-400">
```